core/src/documentation/ide-vscode-setup.html

```diff
@@ -0,0 +1,121 @@
+            embedded Git control, syntax highlighting, intelligent code completion, snippets, and code refactoring. 
+            In part thanks to the huge ecosystem of extensions that enhance and enrich the experience in VS Code, 
+            turning it into a super smart text editor, fast and lightweight. this guide focuses on the bare minimum that 
+            is needed to get started Spark source code with VSCode.
+        </p>
+
+        <h2>Prepare your environment</h2>
+        <p>
+            The rest of this guide will assume you have at least those parts installed:
+            <ul>
+                <li><b>JAVA Development Kit (version 8)</b> </li>
+                <li><b>Maven</b></li> 
+                <li><b>Copy of the  Spark source code available locally </b></li>
+                <li><b>Visual Studio Code</b></li>     
+            </ul>
+        <p>
+            Start by Adding Java language support to VS Code by installing the popular minimum requirement  Java extensions.
+            There is a Java Extension Pack, which contains the most popular extensions for most Java developers: 
+        </p>
+        <p>
+            Launch your VSCode and Bring up the Extensions view by clicking on the <b>Extensions</b> icon in the Activity Bar
+            on the side of VS Code or  <b>View > Extensions</b> then write the name of extension <b>Java Extension Pack</b> and 
+            then clique on install link to install all extension for Pack.
+        </p>
+        <figure>
+            <a href="images/VSCode-1.png"><img src="images/VSCode-1.png" alt="Installing Java Extension Pack to VSCode."></a>
+	    <figcaption>Installing Java Extension Pack to VSCode.</figcaption>            
+        </figure>
+
+        <h2>Open the source code folder</h2>
+        <p>
+            The Spark source code is using an Apache Maven project structure. As VSCode (with Maven Extension) recognizes this structure,
+            opening the source code into an Vscode project is very easy.
+        </p>
+        <p>
+            Select the 'Open' option -- <b>file > open</b> and choose your spark source code folder then clique on the open button. 
+            This will quickly import the Spark source code into  VSCode .
+        </p>
+
+        <figure>
+            <a href="images/VSCode-2.png"><img src="images/VSCode-2.png" alt="Use the 'Open' functionality of VSCode."></a>
+            <a href="images/VSCode-3.png"><img src="images/VSCode-3.png" alt="Select the folder that will open in our VSCode."></a>
+            <a href="images/VSCode-4.png"><img src="images/VSCode-4.png" alt="Overview of project in Visual Studio Code"></a>
+	    <figcaption>Opening Spark Source folder on VSCode.</figcaption>            
+        </figure>
+
+        <p>
+            VSCode will automatically recognize and process the Maven structure of Spark (pom.xml file),and it will download automatically all dependancies.
```

Comment:
no need for comma before and and should be a space there

Reply:
"dependancies" should be spelled as "dependencies"